NPM Builds Fail with CI

Pipeline fails with following log:

npm ERR! The `npm ci` command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "@babel/preset-react": "^7.24.1",
     "babel-loader": "^9.1.3",
     "netlify-cli": "^17.22.1",
-    "sonarqube-scanner": "^3.4.0"
+    "sonarqube-scanner": "^3.4.0",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
   }
 }

